variables.tf

```diff
@@ -477,4 +477,23 @@ variable "gitlab_hpa_max_replicas_webservice" {
   description = "Set the maximum hpa pod replicas for the Gitlab webservice."
   default     = 10
 }
- 
\ No newline at end of file
+
+######################
+# MONITORING SECTION #
+######################
+
+variable "uptime_monitoring_host" {
+  type        = string
+  description = "A hostname to monitor (without protocol, example: 'www.my-site.com')."
+}
```

Comment:
i think we already have the host defined somewhere in other variables, we should reuse it

Reply:
change made.